CRM/Upgrade/Incremental/sql/5.42.alpha1.mysql.tpl

```diff
@@ -1 +1,5 @@
 {* file to handle db changes in 5.42.alpha1 during upgrade *}
+
+ALTER TABLE `civicrm_mapping_field` DROP FOREIGN KEY `FK_civicrm_mapping_field_mapping_id`;
```

Comment:
@seamuslee001 I'm getting confused here - we use `CRM_Core_BAO_SchemaHandler::safeRemoveFK(...)` to drop the index I think but ?? to add one

Reply:
You could use this https://github.com/civicrm/civicrm-core/blob/33f8687f14294e03392b47bec038f747e19ecfda/CRM/Core/BAO/SchemaHandler.php#L256 but still not perfect